### DISCUSSION TITLE
utilizzo shelly

### DISCUSSION THREAD

**Discussion by Firefighter64** [indomus/forum#3081] [GENERALE - Discussioni generiche]
buongiorno a tutti, sono un esordiente totale nel mondo della domotica, avrei necessitò di accendere e spegnere due faresti e ho grosse difficoltà "logistiche" a farlo con un classico interruttore; è fattibile farlo solamente con uno Shelly?
Grazie Dany

**Comment by baldus88** [comment]
Puoi spiegare un po' che difficoltà logistiche hai e cosa vuoi ottenere? anche per capire come indirizzarti magari :)

**Reply by Firefighter64** [reply]
Ciao, le difficoltà "logistiche" sono date dal fatto che ho grosse difficoltà a comandare i faretti con un classico interruttore, pertanto la mia richiesta era mirata a capire se lo shelly funziona anche senza collegargli un classico interruttore. ho fatto delle prove e ci sono riuscito, l'ho collegato senza utilizzare il connettore per l'interruttore e al momento funziona e soddisfa totalmente le mie necessità. Grazie comunque
Dany
